pandas/tests/arrays/test_datetimes.py

```diff
@@ -18,9 +18,8 @@ class TestDatetimeArrayConstructor:
     def test_only_1dim_accepted(self):
         arr = np.array([0, 1, 2, 3], dtype="M8[h]").astype("M8[ns]")
 
-        with pytest.raises(ValueError, match="Only 1-dimensional"):
-            # 2-dim
-            DatetimeArray(arr.reshape(2, 2))
+        # 2-dim allowed for ops compat
+        DatetimeArray(arr.reshape(2, 2))
```

Comment:
Do we are to make any assertion about the `._data` of this result?

Reply:
sure, will update